enroll/transport_http.go

```diff
@@ -0,0 +1,109 @@
+package enroll
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/fullsailor/pkcs7"
+	httptransport "github.com/go-kit/kit/transport/http"
+	"github.com/groob/plist"
+)
+
+type HTTPHandlers struct {
+	EnrollHandler          http.Handler
+	OTAEnrollHandler       http.Handler
+	OTAPhase2Phase3Handler http.Handler
```

Comment:
Could you add a comment here explaining why Phase2 and 3 are in the same handler?